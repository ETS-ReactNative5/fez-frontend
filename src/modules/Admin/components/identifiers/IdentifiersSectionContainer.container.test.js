import { IdentifiersSectionContainer } from './IdentifiersSectionContainer';

jest.mock('context');

function setup(testProps = {}, args = { isShallow: true }) {
    const props = {
        formValues: {
            toJS: jest.fn(() => ({})),
        },
        ...testProps,
    };

    return getElement(IdentifiersSectionContainer, props, args);
}

describe('IdentifiersSectionContainer component', () => {
    it('should render default view', () => {
        const wrapper = setup();
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render disabled view', () => {
        const wrapper = setup({ disabled: true });
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
