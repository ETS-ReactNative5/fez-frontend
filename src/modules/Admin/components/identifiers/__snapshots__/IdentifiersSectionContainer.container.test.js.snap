// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdentifiersSectionContainer component should render default view 1`] = `
<ContextProvider
  value={
    Object {
      "formValues": Object {},
    }
  }
>
  <Memo(IdentifiersSection) />
</ContextProvider>
`;

exports[`IdentifiersSectionContainer component should render disabled view 1`] = `
<ContextProvider
  value={
    Object {
      "formValues": Object {},
    }
  }
>
  <Memo(IdentifiersSection)
    disabled={true}
  />
</ContextProvider>
`;
